### PR TITLE
chore(atomic): add a few chinese localization strings

### PR DESCRIPTION
**Pull request by wmannard**
[QA-979](https://coveord.atlassian.net/browse/QA-979)

### CODE REVIEW

**Comment by wmannard** [comment]
Adding @jeremierobert-coveo who can help verify

File: packages/atomic/src/locales.json [hunk 1]
```diff
@@ -1185,15 +1193,21 @@
   },
   "with-colon": {
     "en": "{{text}}:",
-    "fr": "{{text}}\u00A0:"
+    "fr": "{{text}}\u00A0:",
+    "zh-CN":"{{text}}：",
```
Comment (btaillon) [inline packages/atomic/src/locales.json:1197]: there's a missing space before the string here.
```suggestion
    "zh-CN": "{{text}}：",
```

File: packages/atomic/src/locales.json [hunk 2]
```diff
@@ -563,7 +565,9 @@
   },
   "cancel": {
     "en": "Cancel",
-    "fr": "Annuler"
+    "fr": "Annuler",
+    "zh-CN":"取消",
```
Comment (btaillon) [inline packages/atomic/src/locales.json:569]:
```suggestion
    "zh-CN": "取消",
```

File: packages/atomic/src/locales.json [hunk 3]
```diff
@@ -969,11 +973,15 @@
   },
   "min": {
     "en": "Min",
-    "fr": "Min"
+    "fr": "Min",
+    "zh-CN":"最小值",
```
Comment (btaillon) [inline packages/atomic/src/locales.json:977]:
```suggestion
    "zh-CN": "最小值",
```

**Review by btaillon — APPROVED** [review]
TIL that

File: packages/atomic/src/locales.json [hunk 4]
```diff
@@ -203,7 +203,9 @@
   },
   "page-number": {
     "en": "Page {{page}}",
-    "fr": "Page {{page}}"
+    "fr": "Page {{page}}",
+    "zh-CN":"第{{page}}页",
+    "zh-TW":"第{{page}}頁"
```
Comment (jeremierobert-coveo) [inline packages/atomic/src/locales.json:208]: +1

**Review by jeremierobert-coveo — APPROVED** [review]

**Comment by jeremierobert-coveo** [comment]
> TIL that there's characters that look like the semicolon and parenthesis we're used to but also have a space combined with them.

Welcome to the full-width variations https://www.compart.com/en/unicode/block/U+FF00 😛.

**Review by olamothe — APPROVED** [review]
Nice ! 😂

**Pull request merged**